verification/helpers.go

```diff
@@ -69,6 +69,15 @@ func isPresent(val string, values []string) bool {
 	return false
 }
 
+func isPresentAny(val interface{}, values []interface{}) bool {
```

Comment:
qq: can we get rid of function defined at line 63?

Reply:
> qq: can we get rid of function defined at line 63?

the function at line 63 is used in other places such as policy.go. For example, `isPresent(wildcard, statement.RegistryScopes)`.

Reply:
Can we use generics like 
```go
func isPresent[T comparable](val T, values []T) bool {
	for _, v := range values {
		if v == val {
			return true
		}
	}
	return false
}
```

Reply:
LoL.
```
any does not implement comparable
```